gameserver/src/main/proto/game.proto

```diff
@@ -5,28 +5,73 @@ option java_outer_classname = "GameMessages";
 
 message ServerBoundMessage {
   oneof payload {
-    ServerBoundPlayerJoin player_join = 1;
-    ServerBoundPlayerGuessWord guess_word = 2;
+    ServerBoundJoinGame join_game = 1;
+    ServerBoundStartGame start_game = 2;
+    ServerBoundLeaveGame leave_game = 3;
+    ServerBoundPlayerGuess player_guess = 4;
   }
 }
 
+// ServerBound messages
+message ServerBoundJoinGame {
+  string player_id = 1;
```

Comment:
We actually do not need to include player_id in any of these (if the player_id is meant to be that of the one who is sending the message)
This is because I found out that we can just set an attribute of the socket session itself with each player to have their ID.

You can delete player_id from all of these I think. It is okay for some of them to be empty.

Reply:
I resolved all of the errors -- note that some of the fields are now empty without player_id, which I am guessing is okay, as the message itself is self-explanatory